apps/passport-server/src/services/generic-issuance/genericIssuanceService.ts

```diff
@@ -654,6 +677,14 @@ export class GenericIssuanceService {
           url: f.feedUrl
         })),
         latestAtoms: latestAtoms,
+        latestConsumers: latestConsumers
```

Comment:
should probs be undefined if semaphore group capability is not present on the pipeline